src/Twig/Extension/UnicodeString.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Symfony\Component\String\Exception\ExceptionInterface;
+use Symfony\Component\String\UnicodeString as DecoratedUnicodeString;
+
+/**
+ * Decorates `Symfony\Component\String\UnicodeString` in order to provide the `$cut`
+ * argument for `truncate()`. This class must be removed when the component ships
+ * this feature.
+ *
+ * @see https://github.com/symfony/symfony/pull/35649
+ *
+ * @throws ExceptionInterface
+ *
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+final class UnicodeString
+{
+    private $unicodeString;
+
+    public function __construct(string $string = '')
+    {
+        $this->unicodeString = new DecoratedUnicodeString($string);
+    }
+
+    public function __call($name, $arguments)
```

Comment:
Is this method necessary?

Reply:
Yes, this method provides all the other public methods than `truncate()`.

Reply:
I thought the `truncate` method was returning a `string`, I've seen now that is a `UnicodeString `